main/acle.md

```diff
@@ -9382,6 +9415,2348 @@ are named after. All of the functions have external linkage.
+  
+  
+  // Variants are also available for _za64_u64 and _za64_f64
+  __attribute__((arm_streaming, arm_shared_za, arm_preserves_za))
+  svint64x4_t svread_za64_s64_vg1x4(uint32_t slice_base,
+                                    uint64_t slice_offset);
+  
+  
+  // Variants are also available for _za8[_u8], _za16[_s16], _za16[_u16],
+  // _za16[_f16], _za16[_bf16], _za32[_s32], _za32[_u32], _za32[_f32],
+  // _za64[_s64], _za64[_u64] and _za64_[f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_hor_za8[_s8]_vg2(uint64_t tile, uint32_t slice_base,
+                                uint64_t slice_x2_offset, svint8x2_t zn);
+  
+  
+  // Variants are also available for _za8[_u8], _za16[_s16], _za16[_u16],
+  // _za16[_f16], _za16[_bf16], _za32[_s32], _za32[_u32], _za32[_f32],
+  // _za64[_s64], _za64[_u64] and _za64_[f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_hor_za8[_s8]_vg4(uint64_t tile, uint32_t slice_base,
+                                uint64_t slice_x4_offset, svint8x4_t zn);
+  
+  
+  // Variants are also available for _za8[_u8], _za16[_s16], _za16[_u16],
+  // _za16[_f16], _za16[_bf16], _za32[_s32], _za32[_u32], _za32[_f32],
+  // _za64[_s64], _za64[_u64] and _za64_[f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_ver_za8[_s8]_vg2(uint64_t tile, uint32_t slice_base,
+                                uint64_t slice_x2_offset, svint8x2_t zn);
+  
+  
+  // Variants are also available for _za8[_u8], _za16[_s16], _za16[_u16],
+  // _za16[_f16], _za16[_bf16], _za32[_s32], _za32[_u32], _za32[_f32],
+  // _za64[_s64], _za64[_u64] and _za64_[f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_ver_za8[_s8]_vg4(uint64_t tile, uint32_t slice_base,
+                                uint64_t slice_x4_offset, svint8x4_t zn);
+  
+  
+  // Variants are also available for _za64[_u64] and _za64[_f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_za64[_s64]_vg1x2(uint32_t slice_base, uint64_t slice_offset,
+                                svint64x2_t zn);
+  
+  
+  // Variants are also available for _za64[_u64] and _za64[_f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_za64[_s64]_vg1x4(uint32_t slice_base, uint64_t slice_offset,
+                                svint64x4_t zn);
+  
+  ```
+
+#### PTRUE
+
+Initialise predicate-as-counter to all active
+
+``` c
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svptrue_c8();
+  
+  ```
+
+#### PEXT
+
+Transform a predicate-as-counter to a predicate (pair).
+
+``` c
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svbool_t svpext_lane_c8(svcount_t pnn, uint64_t imm);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svboolx2_t svpext_lane_c8_x2(svcount_t pnn, uint64_t imm);
+  
+  ```
+
+#### PSEL
```

Comment:
I believe this instructions is only available for SME:
https://developer.arm.com/documentation/ddi0602/2022-09/SVE-Instructions/PSEL--Predicate-select-between-predicate-register-or-all-false-?lang=en

Reply:
Well spotted, I'll remove them. I will keep the `svcount_t` variant of psel, because `svcount_t` was only added in SME2.

Reply:
Addressed in https://github.com/ARM-software/acle/pull/217/commits/18c5da294a88c3a4fbb4c8d3301c5f71293f1aff